www/js/diary/list.js

```diff
@@ -76,10 +76,26 @@ angular.module('emission.main.diary.list',['ui-leaflet', 'nvd3ChartDirectives',
       return ($scope.dark_theme)? "ion-view-background-dark" : "ion-view-background"; 
     }
     $scope.datePickerClass = function() {
-
     }
     $scope.listCardClass = function() {
-      return ($scope.dark_theme)? "list card list-card-dark" : "list card list-card"; 
+      if ($window.screen.width <= 320) {
```

Comment:
Again, this looks like the three sizes for iphone. What happens about android phones?